cclib/method/volume.py

```diff
@@ -365,3 +367,68 @@ def electrondensity(ccdata, volume, mocoeffslist):
         edens.data *= 2
         return edens
 
+
+def read_from_cube(filepath):
+    """Read data from cube files and construct volume object
+    
+       Specification of cube file format is described in http://paulbourke.net/dataformats/cube/
+       
+    Input:
+        filepath -- path to the cube file
+        
+    Output:
+        vol -- Volume object filled with data from cube file
+    """
+
+    # Cube files are written in bohrs
+    def convert(x):
+        return convertor(x, "bohr", "Angstrom")

```

Comment:
`covertor(x, 'bohr', 'Angstrom')` isn't much longer and is more explicit when used. could this be changed back

Reply:
I've modified this into a full call. Thanks!